src/main/java/org/folio/fqm/service/CrossTenantQueryService.java

```diff
@@ -48,6 +49,9 @@ public List<String> getTenantsToQuery(EntityType entityType, boolean forceCrossT
           .read("consortia[0].id");
       } else {
         log.debug("Tenant {} is not central tenant. Running intra-tenant query.", executionContext.getTenantId());
+        if (COMPOSITE_INSTANCES_ID.equals(entityType.getId())) {
```

Comment:
We should probably create a ticket to replace this with an entity type property of some kind, since it feels kind of gross to hard-code the ID

Reply:
Also, a comment here to explain the weirdness would be super useful later on when reading this code

Reply:
We could, but it's such a specific thing that seems like it will only apply to composite instances. I'm not aware of anything else that needs to retrieve shared records from the central tenant, so to me it feels like the property would clutter things more (especially since we already have a bunch that rarely get used).